runner/runner/main.py

```diff
@@ -116,21 +115,48 @@ def load_native(config: Mapping[str, Any]) -> None:
         ],
         desc="Building plugins",
     )
-    command_str = config["loader"].get("command", "%a")
-    main_cmd_lst = [str(runtime_exe_path), *map(str, plugin_paths)]
-    command_lst_sbst = list(
+    actual_cmd_str = config["loader"].get("command", "$cmd")
+    illixr_cmd_list = [str(runtime_exe_path), *map(str, plugin_paths)]
+    env_override = dict(
+        ILLIXR_DATA=str(data_path),
+        ILLIXR_DEMO_DATA=str(demo_data_path),
+        KIMERA_ROOT=config["loader"]["kimera_path"],
+    )
+    env_list = [
+        f"{shlex.quote(var)}={shlex.quote(val)}" for var, val in env_override.items()
+    ]
+    actual_cmd_list = list(
         flatten1(
             replace_all(
-                unflatten(shlex.split(command_str)),
-                {("%a",): main_cmd_lst, ("%b",): [shlex.quote(shlex.join(main_cmd_lst))]},
```

Comment:
This change conflicts with the documentation in #215. It does not induce an ordering; changes are required in whichever is not merged first.

`%a` -> `$cmd` or `$env_cmd`
`%b` -> `$quoted_cmd`

Reply:
This PR can be merged first. Which page/section is the conflict in?

Reply:
`docs/building_illixr.md`

- `$cmd` is the pure cmd, argument for argument.
- `$quoted_cmd` is the cmd as one argument.
- `$env_cmd` is like `$cmd`, but it sets env-vars explicitly `env A=B C=D ... $cmd`. The runner already sets the env-vars in the process of `command`, but this is useful if you want to run outside the runner (you'll need the env-vars explicitly set).